.envrc

```diff
@@ -0,0 +1 @@
+eval "$(lorri direnv)"
```

Comment:
what is this file?

Wht happens if there is `direnv` user who isn't nix user?

Reply:
This is how [`lorri`](https://github.com/target/lorri) is intended to be used. The first time a `direnv` user enters the directory, they will be prompted to run `direnv allow`. If they do and wonder why they get an error, they can look at that file and figure out that they need lorri. And if they don't figure it out, they run `direnv deny` or write their own `.envrc`.

Reply:
Ok. add that as a comment. I don't want people to think I need that `.envrc` file to exist at all.

(I'm sorry, but Nix is solving problems I don't have, but it potentially brings problems I didn't have before).

A compromise is to add `.envrc` to the `.gitignore`.

Reply:
Sorry, I was unclear.

Let's just remove that file. Add it to `.gitignore` if you need it locally.